openedx_events/enterprise/data.py

```diff
@@ -8,9 +8,9 @@
 
 
 @attr.s(frozen=True)
-class LicenseLifecycle:
+class SubscriptionLicenseData:
     """
-    Event to track changes to subscriptsion.License records.
+    Event to track changes to subscription license records.
```

Comment:
The following suggestion was in the last review based on the learning data object docstring pattern.
```suggestion
    Attributes defined for Open edX Subscription License object.
```
Note: Not sure how and if this should be updated if we are adding an attribute for the lifecycle stage, since this is no longer just representing a license.